index.html

```diff
@@ -1,11 +1,28 @@
-<!DOCTYPE html>
-<html lang="en">
+<html>
+<!-- Text between angle brackets is an HTML tag and is not displayed.
+Most tags, such as the HTML and /HTML tags that surround the contents of
+a page, come in pairs; some tags, like HR, for a horizontal rule, stand 
+alone. Comments, such as the text you're reading, are not displayed when
+the Web page is shown. The information between the HEAD and /HEAD tags is 
+not displayed. The information between the BODY and /BODY tags is displayed.-->
 <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>crap portfolio</title>
+<title>Enter a title, displayed at the top of the window.</title>
 </head>
+<!-- The information between the BODY and /BODY tags is displayed.-->
 <body>
-    
+<h1>Enter the main heading, usually the same as the title.</h1>
+<p>Be <b>bold</b> in stating your key points. Put them in a list: </p>
+<ul>
+<li>The first item in your list</li>
+<li>The second item; <i>italicize</i> key words</li>
+</ul>
```

Comment:
I don't like unordered list but whatevs
```suggestion
<ol>
<li>The first item in your list</li>
<li>The second item; <i>italicize</i> key words</li>
</ol>
```